x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/test_query_row.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiButtonEmpty, EuiFormRow, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useTestQuery } from './use_test_query';
+
+export function TestQueryRow({
+  fetch,
+  hasValidationErrors,
+}: {
+  fetch: () => Promise<{ nrOfDocs: number; timeWindow: string }>;
+  hasValidationErrors: boolean;
+}) {
+  const { onTestQuery, testQueryResult, testQueryError } = useTestQuery(fetch);
+
+  return (
+    <>
+      <EuiFormRow>
+        <EuiButtonEmpty
+          data-test-subj="testQuery"
+          color={'primary'}
```

Comment:
nit: lets follow single style with double quotes for string props.

Reply:
very true, just copied code here 👍 , fixed